StreamChatClient_v3/Models/ChannelModel.swift

```diff
@@ -57,6 +57,8 @@ public struct ChannelModel<ExtraData: ExtraDataTypes> {
     /// Checks if the channel is watching by the client.
     public let isWatched: Bool
     
+    public let messages: [MessageModel<ExtraData>]
```

Comment:
I'd call this `latestMessages` or something like that to signalize it's not all messages

Reply:
The same should be for watchers.